### PR TITLE
DOCSP-23037 supported instance types

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-23037-supported-instance-types/reference/mongosync/)

[JIRA](https://jira.mongodb.org/browse/DOCSP-23037)

### CODE REVIEW

File: source/reference/api/pause.txt [hunk 1]
```diff
@@ -4,6 +4,8 @@
 ``pause``
 =========
 
+.. include:: /includes/preview-warning
```
Comment (davemungo) [inline source/reference/api/pause.txt:7]: Not related to this Jira - update / clean up to incorporate recent changes

File: source/reference/api/resume.txt [hunk 2]
```diff
@@ -4,6 +4,8 @@
 ``resume``
 ==========
 
+.. include:: /includes/preview-warning
```
Comment (davemungo) [inline source/reference/api/resume.txt:7]: Not related to this Jira - update / clean up to incorporate recent changes

File: source/reference/mongosync.txt [hunk 3]
```diff
@@ -19,11 +19,12 @@
 Description
 -----------
 
-The ``mongosync`` binary is the primary process used in Cluster-to-Cluster Sync.
-It migrates data from one cluster to another and can keep the clusters
-in continuous sync.
+The ``mongosync`` binary is the primary process used in
+{+c2c-product-name+}. ``mongosync`` migrates data from one cluster to
```
Comment (davemungo) [inline source/reference/mongosync.txt:23]: Not related to this Jira - update / clean up to incorporate recent changes

File: source/reference/mongosync.txt [hunk 4]
```diff
@@ -34,39 +35,44 @@ Global Options
 .. option:: --cluster0 <URI>
 
    Sets the :ref:`connection URI <mongodb-uri>` for the first cluster.
-   The first cluster can serve as either the source cluster or the destination 
-   cluster in the sync process.
+   The first cluster can serve as either the source cluster or the
```
Comment (davemungo) [inline source/reference/mongosync.txt:38]: The update for this Jira

File: source/reference/mongosync.txt [hunk 5]
```diff
@@ -34,39 +35,44 @@ Global Options
...
    For more information on connecting ``mongosync``, see
    :ref:`Connections <c2c-connecting>`.
 
-
 .. option:: --cluster1 <URI>
 
    Sets the :ref:`connection URI <mongodb-uri>` for the second cluster.
-   The second cluster can serve as either the source cluster or the destination 
-   cluster in the sync process.
+   The second cluster can serve as either the source cluster or the
```
Comment (davemungo) [inline source/reference/mongosync.txt:55]: The update for this Jira

File: source/reference/mongosync.txt [hunk 6]
```diff
@@ -34,39 +35,44 @@ Global Options
...
    :ref:`Connections <c2c-connecting>`.
 
-
 .. option:: --verbosity <level>
 
    *Default*: ``INFO``
 
-   Sets the verbosity level to use in log messages.  Cluster-to-Cluster Sync logs
-   all messages at the specified level as well as any messages at less severe levels.
+   Sets the verbosity level to use in log messages.
```
Comment (davemungo) [inline source/reference/mongosync.txt:73]: Not related to this Jira - update / clean up to incorporate recent changes

File: source/reference/mongosync.txt [hunk 7]
```diff
@@ -78,46 +84,43 @@ Global Options
    - ``FATAL``
    - ``PANIC``
 
-
 .. option:: --logPath <DIR>
 
-   Sets the path to the log directory. Cluster-to-Cluster Sync writes logs to
-   files in this directory.
-
+   Sets the path to the log directory. {+c2c-product-name+} writes logs
```
Comment (davemungo) [inline source/reference/mongosync.txt:89]: Not related to this Jira - update / clean up to incorporate recent changes

File: source/reference/mongosync.txt [hunk 8]
```diff
@@ -78,46 +84,43 @@ Global Options
...
+   to files in this directory.
 
 .. option:: --port
 
    *Default*: ``27182``
 
-   Sets the port used by the HTTP server for the Cluster-to-Cluster Sync REST API.
-
+   Sets the port used by the HTTP server for the {+c2c-product-name+}
```
Comment (davemungo) [inline source/reference/mongosync.txt:96]: Not related to this Jira - update / clean up to incorporate recent changes

File: source/reference/mongosync.txt [hunk 9]
```diff
@@ -78,46 +84,43 @@ Global Options
...
 
 .. option:: --id <ID>
 
    Sets an identifier for the ``mongosync`` instance.
 
-   Use this option when running multiple instances of ``mongosync`` on a sharded
-   cluster, to synchronize the shards individually.  The ``--id`` option must
-   correspond to the shard ID of the shard it
-   syncs.  To find the shard ID, use the :dbcommand:`listShards` command.
+   Use this option when running multiple instances of ``mongosync`` on
```
Comment (davemungo) [inline source/reference/mongosync.txt:103]: Not related to this Jira - update / clean up to incorporate recent changes

File: source/reference/mongosync.txt [hunk 10]
```diff
@@ -126,16 +129,16 @@ Behavior
 Configuration File
 ~~~~~~~~~~~~~~~~~~
 
-Options for ``mongosync`` can be set in a YAML configuration file, specified
-using the :option:`--config` option.  For example:
+Options for ``mongosync`` can be set in a YAML configuration file. Use
```
Comment (davemungo) [inline source/reference/mongosync.txt:132]: Not related to this Jira - update / clean up to incorporate recent changes

**Review by davemungo — COMMENTED** [review]
The edits for this update are marked in the comments. 

Since this is a fix-up ticket, I also updated and cleaned up to incorporate recent updates to the site. Those updates are also marked accordingly.

File: source/reference/mongosync.txt (hunk 4 above)
Comment (sarah-olson-mongodb) [inline source/reference/mongosync.txt:38]: Maybe too many clusters in this sentence. Consider something like:
You can specify the first cluster as either the source or destination cluster for the sync, using the call to the ...

File: source/reference/mongosync.txt (hunk 5 above)
Comment (sarah-olson-mongodb) [inline source/reference/mongosync.txt:55]: Similar comment as above. Consider simplifying.

**Review by sarah-olson-mongodb — APPROVED** [review]
Hi Dave. This looks good! Thank you. I added a couple minor suggestions for your consideration.
LGTM

File: source/reference/mongosync.txt [hunk 11]
```diff
@@ -34,39 +35,44 @@ Global Options
...
+   The second cluster can serve as either the source cluster or the
+   destination cluster in the sync process. Designate the source and
+   destination clusters in the call to the :ref:`c2c-api-start` API
+   endpoint.
 
    .. note::
 
-      The cluster must use MongoDB 6.0 or later.
-
+      - The cluster must use MongoDB 6.0 or later.
```
[the remark this answers is not in the document]
Comment (davemungo) [reply source/reference/mongosync.txt]: Done. 

I moved it to the top of the page for prominence.

**Review by davemungo — COMMENTED** [review]
Thanks @sarah-olson-mongodb  sending this on to external review next

File: source/reference/mongosync.txt [hunk 12]
```diff
@@ -126,16 +121,16 @@ Behavior
...
-using the :option:`--config` option.  For example:
+Options for ``mongosync`` can be set in a YAML configuration file. Use
+the :option:`--config` option. To specify the path to the configuration
+file. For example:
 
 .. code-block:: yaml
 
    # Cluster Configuration
    cluster0: "mongodb://192.0.2.10:27017,192.0.2.11:27017,192.0.2.12:27017"
    cluster1: "mongodb://192.0.2.20:27017,192.0.2.21:27017,192.0.2.22:27017"
```
Comment (alan-zheng-mdb) [inline source/reference/mongosync.txt:132]: just tested using a config file, I don't think the quotes are required

File: source/reference/mongosync.txt [hunk 13]
```diff
@@ -19,11 +19,18 @@
...
+{+c2c-product-name+}. ``mongosync`` migrates data from one cluster to
+another and can keep the clusters in continuous sync. 
 
-This document provides a complete overview of all command-line options.
+This document provides a complete overview of the ``mongosync`` command
+line options.
+
+.. note::
+
+   - The cluster must use MongoDB 6.0 or later.
```
Comment (alan-zheng-mdb) [inline source/reference/mongosync.txt:31]: Both the source and destination cluster must use MongoDB 6.0 or later.

File: source/reference/mongosync.txt [hunk 14]
```diff
@@ -19,11 +19,18 @@
...
+   - ``mongosync`` supports replica sets and sharded clusters.
+   - Standalone MongoDB instances are not supported.
```
Comment (alan-zheng-mdb) [inline source/reference/mongosync.txt:33]: Potentially link this https://www.mongodb.com/docs/manual/tutorial/convert-standalone-to-replica-set/ with the word standalone?

**Review by alan-zheng-mdb — APPROVED** [review]
Thanks Dave. Small fixes, but LGTM

**Review by davemungo — COMMENTED** [review]
Thanks @alan-zheng-mdb  I've updated the content as suggested and will merge.

**Pull request merged**